unity/Assets/Scripts/StretchAgentController.cs

```diff
@@ -13,6 +13,11 @@ public partial class StretchAgentController : PhysicsRemoteFPSAgentController {
         public StretchAgentController(BaseAgentComponent baseAgentComponent, AgentManager agentManager) : base(baseAgentComponent, agentManager) {
         }
 
+        public override void updateImageSynthesis(bool status) {
+            base.updateImageSynthesis(status);
+            agentManager.updateThirdPartyCameraImageSynthesis(true);
```

Comment:
This should get changed to pass `status` to `agentManager.updateThirdPartyCameraImageSynthesis`

Reply:
It would also make sense to add a comment above the `updateThirdPartyCameraImageSynthesis` line explaining why this is necessary since it wouldn't be obvious that the secondary camera is assumed to have been added.